savoirfairelinux/ringme.js
# updated ringme.js landing page

## Fixes #24 by adding theme (it's not super curly, hopefully that's ok)

### Changes proposed in this pull request:

* added github logo for github project link
* added stylesheet
* modified index.html

### Status


### How to verify this change

1. Check out the `gh-pages` branch.
2. Open `index.html` in a web browser.

### Additional notes

Not sure if curly was just a joke or if you guys actually wanted curls, let me know and I can modify the landing page to suit your needs. :)
